google/ggrc-core
# GGRC-6768 ACL changes to remove evidence for assessment

# Issue description

Currently **global creator**/**global reader** user with **auditor** or assessment **creator**/**assignee**/**verifier** role can not remove (unmap) evidence file/url from assessment, but it should be permitted.

# Steps to test the changes

1. Log ggrc as admin user;
2. Create program, control, audit;
3. Open audit page, create assessment;
4. Assign **global creator**/**global reader** user as **auditor** to audit or **creator**/**assignee**/**verifier** to assessment;
5. Log as **global creator**/**global reader** user and open audit page;
6. Attach evidence file/url to assessment;
7. Try to remove evidence file or url;

Expected result: evidence file/url is successfully removed.

# Solution description

Solutions includes updating propagation tree for **auditors**, **assignees**, **creators** and **verifiers** roles so they can remove `Evidence` objects.

# Sanity checklist


# Migration checklist

# PR Review checklist
